pipeline/metadata/6.0.yaml

```diff
@@ -772,6 +761,16 @@ pipelines:
             openstack: "conf/inventory/rhel-9-latest.yaml"
           metadata:
             - rgw
+        - name: "Tier-2 cephfs upgrade 5-3 to 6-0"
+          execution_time: ""
```

Comment:
If you have an execution for this, could you please update the execution time here.

Reply:
Done manasa